pkgs/tools/networking/dnscrypt-proxy/2.x/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildGoPackage, fetchFromGitHub, go }:
+
+buildGoPackage rec {
+  name = "dnscrypt-proxy2-${version}";
+  version = "2.0.15";
+
+  goPackagePath = "github.com/jedisct1/dnscrypt-proxy";
+
+  src = fetchFromGitHub {
+    owner = "jedisct1";
+    repo = "dnscrypt-proxy";
+    rev = "${version}";
+    sha256 = "0iwvndk1h550zmwhwablb0smv9n2l51hqbmzj354mcgi6frnx819";
+  };
+
+  enableParallelBuilding = true;
```

Comment:
no need to set this as it's default https://github.com/NixOS/nixpkgs/blob/92609ec9ae725da937cdca7a4dcca1129a8bb919/pkgs/development/go-modules/generic/default.nix#L7